test/compress/properties.js

```diff
@@ -2486,3 +2486,13 @@ mangle_undeclared_properties: {
         var r={o:function(){return a.t()}};
     }
 }
+
+keep_numbers_in_properties_as_is: {
+    beautify = {
+        keep_numbers: true
+    }
+    input: {
+        var Foo = { 1000000: 80000000000 }
+    }
+    expect_exact: "var Foo={1000000:80000000000};"
+}
```

Comment:
Could you put this test in `test/compress/numbers.js`? It makes a lot more sense there!